content/docs/use-cases/ci-cd-for-machine-learning.md

```diff
@@ -0,0 +1,89 @@
+# Continuous Integration and Deployment for Machine Learning
+
+## DevOps
+
+Applying DevOps methodologies to machine learning (MLOps) and data management
+(DataOps) is increasingly common. This means resource orchestration
+(provisioning servers), testing, and deployment to production (CI/CD --
+continuous integration and continuous delivery), as well as monitoring &
+feedback.
+
+![](https://static.iterative.ai/img/ml-vs-cicd.png) _Traditional machine
+learning (ML) meets CI/CD_
+
+The main benefits of CI/CD for ML are detailed below.
+
+### Quality Assurance (QA)
```

Comment:
This heading is a generic topic, but the next one is a product. Can we make them consistent since the paragraphs under them look so similar? Maybe something like `Quality Assurance [with DVC]`/`Cloud Provisioning [with CML]`  (not sure if including products is needed).

Reply:
I actually intended the next heading to also be interpreted as a concept (if you look at the content ou see it makes no mention of the product). I agree it's confusing that we happen to have a product with the same name as the concept (it's actually the # 1 cause of confusion for CML-the-product users. Many people claim to be using CML-the-product when they're actually only using CML-the-concept-but-reimplementing-the-product - this even includes one of the speakers at the last meetup!)

Reply:
removed headings now